### DISCUSSION TITLE
What can cause lncli getinfo to be so slow to the point it hasn't return anything after 5 minutes of waiting?

### DISCUSSION THREAD

**Discussion by FeatureSpitter** [lightningnetwork/lnd#8008] [Troubleshooting] [answered]
There are no errors in the logs, other commands like walletbalance and getnetworkinfo return (they take a few seconds tho), but getinfo and listpeers and maybe others take forever to return... no error in the logs. I am using 0.17 RC3 on raspberry pi 4 8GB. The CPU is at 1% or less...

Logs: https://pastebin.com/W10vaKgF

Related issue: https://github.com/lightningnetwork/lnd/issues/8009

**Comment by guggero** [comment, marked as answer]
Please don't cross post.
